test/gpu/native/localeNodeId/localeIdOnHost.chpl

```diff
@@ -0,0 +1,10 @@
+use BlockDist;
```

Comment:
Wait, did this test not already work? (not saying it's not a good test to have, just want to make sure I understand what this PR is doing).

Reply:
Yep :( That's why I kinda wanted to rush the patch. https://github.com/chapel-lang/chapel/issues/21977

Reply:
Got it, this issue makes sense now.